src/main/util/solveQuadraticEquation.js

```diff
@@ -16,7 +16,7 @@ export function solveQuadraticEquation (a0 : number, a1 : number, a2 : number, s
   else {
     const disc = a1*a1 - 4*a0*a2;
     if (disc < 0) {
-      return false; // non-real solutions
+      return null; // non-real solutions
```

Comment:
Fixed a lingering issue with the quadratic equation solver returning "false" instead of "null". This fixes some corner collision issues among other things.